04.ls/wc.rb

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options, paths = receive_command_argument
+  paths.empty? ? output(count_stdin, options) : output(calculate_file_text(paths), options)
+end
+
+def receive_command_argument

```

Comment:
このメソッド名でもだいたいやろうとしていることはわかりますが、この処理はOptionParserを使って、コマンドライン引数（＝オプション）をパースしているので、
```suggestion
def parse_options
```

みたいな名前にするとよりシンプルになってよいかもしれません

Reply:
ネーミングセンスを磨きます。
修正しました